test/specs/controller.bar.tests.js

```diff
@@ -823,10 +823,10 @@ describe('Bar controller tests', function() {
 		var meta0 = chart.getDatasetMeta(0);
 
 		[
-			{b: 290, w: 83, x: 86, y: 161},
-			{b: 290, w: 83, x: 202, y: 419},
-			{b: 290, w: 83, x: 318, y: 161},
-			{b: 290, w: 83, x: 434, y: 419}
+			{b: 290, w: 83 / 2, x: 63, y: 161},
```

Comment:
I think that was a bug introduced by #3563, it should be half the width because the x scale is not stacked, right?